contract/r/gnoswap/gov/staker/reward_calculation.gno

```diff
@@ -0,0 +1,334 @@
+package staker
+
+import (
+	"std"
+
+	en "gno.land/r/gnoswap/v1/emission"
+	pf "gno.land/r/gnoswap/v1/protocol_fee"
+
+	"gno.land/p/gnoswap/consts"
+	"gno.land/r/gnoswap/v1/common"
+
+	"gno.land/p/demo/avl"
+	ufmt "gno.land/p/demo/ufmt"
+	u256 "gno.land/p/gnoswap/uint256"
+)
+
+var (
+	currentGNSBalance         uint64            = 0
+	currentProtocolFeeBalance map[string]uint64 = make(map[string]uint64)
+)
+
+func getCurrentBalance() uint64 {
+	gotGnsForEmission = en.GetDistributedToGovStaker()
+	en.ClearDistributedToGovStaker()
```

Comment:
Why?

Reply:
[Comments](https://github.com/gnoswap-labs/gnoswap/pull/488/files#diff-3323c467dc6783864a303826314ed7ef18e507f0ca277844bccc2975d0b5c679R22-R23) to explain.